Makefile

```diff
@@ -43,14 +44,18 @@ $(GEN_CRD_API_REF_DOCS): $(TOOLS_DIR)/go.mod ## Build gen-crd-api-ref-docs from
 # TODO: Check again later if we can migrate this there
 OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.12.0
 $(OPERATOR_SDK): ## Build operator-sdk from tools folder.
-	curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${GOOS}_${GOARCH}
-	gpg --keyserver keyserver.ubuntu.com --recv-keys 052996E2A20B5C7E
-	curl -LO ${OPERATOR_SDK_DL_URL}/checksums.txt
-	curl -LO ${OPERATOR_SDK_DL_URL}/checksums.txt.asc
-	gpg -u "Operator SDK (release) <cncf-operator-sdk@cncf.io>" --verify checksums.txt.asc
-	grep operator-sdk_${GOOS}_${GOARCH} checksums.txt | sha256sum -c -
-	chmod +x operator-sdk_${GOOS}_${GOARCH} && sudo mv operator-sdk_${GOOS}_${GOARCH} ${OPERATOR_SDK}
-	rm checksums.txt checksums.txt.asc
+	bash -c "set +x; \
+		$(eval TMPDIR=$(shell mktemp -d)) \
+		trap 'rm -rf $(TMPDIR)' EXIT \
+		&& cd $(TMPDIR) \
+		&& wget ${OPERATOR_SDK_DL_URL}/operator-sdk_$(GOOS)_$(GOARCH) \
+		&& wget ${OPERATOR_SDK_DL_URL}/checksums.txt \
+		&& wget ${OPERATOR_SDK_DL_URL}/checksums.txt.asc \
+		&& gpg --keyserver keyserver.ubuntu.com --recv-keys 052996E2A20B5C7E \
+		&& gpg -u 'Operator SDK (release) <cncf-operator-sdk@cncf.io>' --verify $(TMPDIR)/checksums.txt.asc \
+		&& grep operator-sdk_${GOOS}_${GOARCH} checksums.txt | sha256sum -c - \
+		&& chmod 750 operator-sdk_$(GOOS)_$(GOARCH) \
+		&& sudo mv operator-sdk_$(GOOS)_$(GOARCH) $(ROOT_DIR)/${OPERATOR_SDK}"
```

Comment:
Can we do without `sudo`? Some Linux distributions do not have it out of the box?

Reply:
yes we can do without; i removed it